mobile-bare/.buckconfig

```diff
@@ -0,0 +1,6 @@
+
```

Comment:
Pains me to add unused config files for expokit's dependencies, are they necessary for the project to run/compile? :stuck_out_tongue: (noticed this one + the watchman config are empty, plus some other files that are blank except for comments)

Reply:
> Pains me to add unused config files for expokit's dependencies, are they necessary for the project to run/compile?

I think this file is generated even in a regular react native project. Why do you think it's unused?